problem-spring-web/src/test/java/org/zalando/problem/spring/web/advice/security/SecurityAdviceTraitTest.java

```diff
@@ -45,12 +49,16 @@
 @WebAppConfiguration
 final class SecurityAdviceTraitTest {
 
+    @BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem-spring-web/pullRequest?prid=4540504)

Reply:
I disabled that rule just now